hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/SequenceIdAccounting.java

```diff
@@ -250,7 +250,12 @@ void updateStore(byte[] encodedRegionName, byte[] familyName, Long sequenceId,
    */
   private static long getLowestSequenceId(Map<?, Long> sequenceids) {
     long lowest = HConstants.NO_SEQNUM;
-    for (Long sid: sequenceids.values()) {
+    for (Map.Entry<? , Long> entry : sequenceids.entrySet()){
+      if (entry.getKey() instanceof ImmutableByteArray && entry.getKey().toString().equals("METAFAMILY")){
```

Comment:
Do we need this extra `entry.getKey() instanceof ImmutableByteArray` check? Wouldn't `entry.getKey().toString().equals("METAFAMILY")` already gives `false` in the cases `entry.getKey()` isn't a `ImmutableByteArray`?

Reply:
entry.getKey() instanceof ImmutableByteArray is  for future,   but it's no need at now , I has deleted it !